### ISSUE TITLE
Laser Scalpel cautery mode does not work for limb replacement when in surgeon switchtools

### ISSUE THREAD

**Issue by D3athrow-Issues** [d3athrow/vgstation13#32447]
(WEB REPORT BY: luigil101 REMOTE: 206.221.180.138:7777)
# Revision
ef016fceffbeaeb4de97e926f3ba7cb0e4ebb892

# Description
Laser Scalpel cautery mode does not work for limb replacement surgery when in surgeon switchtools, though does do so for most other operations
this causes the operation steps to infinitely cycle between scalpel and retractor until you get either a second laser scalpel set to cautery mode, or add an old, obsolete, cautery to your switchtool
# Steps to Reproduce
Try to put a limb/head back on a man
# What you Expected
space age scalpel to cauterize on demand like on the label
# What Actually Happened
around around around it goes, when this limb step will appear, nobody knows

**Comment by Eneocho** [comment]
You can't toggle modes inside a switchtool, so the laser scalpel can only work as a scalpel, switchtools as a whole are very broken, with many interactions being impossible inside one, like butchering or lighting fuel fires with a SAK's knife/lighter. You gotta take em out for them to work properly.

**Comment by luigil101** [comment]
a shame switchtools as a whole are fucky because they're great

> > though does so for most other operations
> 
> huh? you're saying it works as a cautery even though it says scalpel?

Laser scalpels are the upgraded version and have a toggle cautery setting, and they'll act as a cautery in a switchtool automatically for bone/int. bleed surgery no problem
its extremely fucking nice

**Comment by kernikov** [comment]
...You can separate the cautery from a laser scalpel by switching it to cautery and then using a screwdriver on it. You can then add both to the switchtool.

**Comment by luigil101** [comment]
yeah that's a perfectly effective workaround and is what I do now, but the false advertising triggered my tism
